pygamelib/engine.py

```diff
@@ -2505,6 +2505,7 @@ def _obj2ref(obj):
             "name": obj.name,
             "pos": obj.pos,
             "model": obj.model,
+            "sprixel": obj.sprixel,
```

Comment:
Actually the fix is pretty simple: just use `obj.sprixel.serialize()` instead of `obj.sprixel`.